Guidelines for API design PRs (#24796)
fleetdm/fleet: handbook/product-design/README.md

3. Add page names (e.g. "Host details" page) to the user story's title and/or description to help contributors find Figma wireframes for the area of the UI you're making changes to.

> 4. If the story requires API or YAML file changes, open a pull request to the reference docs with the proposed design. Pay attention to existing conventions (URL structure, parameter names, response format) and aim to be consistent. Once your changes are ready, make a PR following these guidelines:

Reviewer: ```suggestion
4. If the story requires API or YAML file changes, open a pull request (PR) to the reference docs with the proposed design. Pay attention to existing conventions (URL structure, parameter names, response format) and aim to be consistent. Your PR should follow these guidelines:
```

Small language tweaks. Up to you.

>   - Make a PR against the docs release branch for the version you expect this feature to be in. Docs release branches are named using the format `docs-vX.X.X`, so if you're designing for Fleet 4.61, you would make a PR to `docs-v4.61.0`.

Reviewer: ```suggestion
  - Make a PR against the docs release branch for the version you expect this feature to be in. Docs release branches are named using the format `docs-vX.X.X`, so if you're designing for Fleet 4.61.0, you would make a PR to `docs-v4.61.0`.
```

Might as well us the entire semantic version here right? Avoid future confusion/questions like what's the difference between 4.61 and 4.61.0?

  - Add a link to the issue in the PR description.
  - Attach the `~api-design` label.
  - Mark the PR ready for review. (Draft PRs do not auto-request reviews.)

>   - After your changes are approved by the API design DRI and the design is settled, merge your changes into the docs release branch.

Reviewer: ```suggestion
  - After your changes are approved by the API design DRI, they will merge your changes into the docs release branch.
```

Instead of leaving it up to the Product Designer to merge, maybe the API design DRI merge in the changes when approved? 

- I think this would mean PRs wouldn't be open as long. API design DRI doesn't have to revisit an open PR just to see that it's already been approved.
- Also, I think it's up to the Product Designer to mark a PR as ready for review when it's ready for review i.e. "settled"
